pro/src/ui-kit/Banners/InternalBanner/InternalBanner.tsx

```diff
@@ -16,7 +15,7 @@ interface IInternalBannerProps extends IBannerLayoutProps {
   >
   linkTitle: string
   subtitle?: string
-  to?: LocationDescriptor
```

Comment:
Oh pourquoi ? :'( c'était pour faire passer un state si besoin, on peut passer une string (c'est inclus dans LocationDescriptor)

Reply:
T'as un cas d'usage pour les bannière interne où on en aurait besoin ? (je suis pas sur de saisir la fonction de LocationDescriptor j'oublie sans doute qq chose)

Reply:
On pourrait imaginer un truc de tracking où tu veux tracker la provenance de l'utilisateur sur une page, et tu pourrais par exemple passer à InternalBanner un `to={{pathname: '/mapage', state: 'lieu'}}` et tu pourrais du coup utiliser la valeur du state là où t'as besoin du tracking
C'est pas un cas qu'on a là maintenant mais les liens internes utilisent des `Link` qui prennent un `LocationDescriptor` pour la props `to` donc je trouve ca dommage de se restreindre à une string alors que ca peut etre bien plus puissant derrière. Et `LocationDescriptor` c'est soit une string soit un objet du coup ca nous coûte pas très cher

Reply:
Ce n'est pas un besoin pour le moment, c'est toujours un peu dangereux d'imaginer et d'anticiper, car le besoin final est souvent un peu différent de celui qu'on avait imaginé, et modifier le code devient plus compliqué que l'écrire au moment où on en a besoin